src/wovn_interceptor.php

```diff
@@ -31,9 +33,13 @@
 $headers->requestOut();
 
 $uri = $headers->getDocumentURI();
-if (!Utils::isFilePathURI($uri, $store) &&
-    !Utils::isIgnoredPath($uri, $store)
-    ) {
+if (!Utils::isIgnoredPath($uri, $store)) {
+    $filePath = wovn_helper_detect_paths(__DIR__ . '/../..', $uri);
```

Comment:
Does this code expect that WOVN.php is on the root dir?
I saw some clients are using WOVN.php with not root dir :cry:

Reply:
Yeah it does assume WOVN.php folder is at document root. Let me change it to make use of _SERVER['DOCUMENT_ROOT']

Reply:
Since we no longer check the request file, but only the response buffer, this is no longer a problem.